### PR TITLE
feat(scripts): improve scripts for running Kovan EMP war games

### PR DESCRIPTION
**Pull request by pemulis**
**Motivation**

Some improvements to the scripts for testing EMPs.

**Summary**

Mostly updates to use the latest ABI version and pass parameters in as arguments rather than hardcoding values for certain scripts. Also adds a default to the `TransferCollateral` and `TransferSynthetic` scripts to sweep the entire token balance from the send account to the recipient account.

**Details**

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [x]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [ ]  Untested

**Issue(s)**

Fixes #XXXX

### CODE REVIEW

File: packages/core/scripts/local/DeployEMP.js [hunk 1]
```diff
@@ -100,11 +104,11 @@ const deployEMP = async callback => {
 
     // Create a new EMP
     let constructorParams = {
-      expirationTimestamp: Math.ceil(Date.now() / 1000) + 60 * 2, // 2 minutes from now
+      expirationTimestamp: expirationTimestamp,
       collateralAddress: collateralToken.address,
       priceFeedIdentifier: priceFeedIdentifier,
-      syntheticName: "Test Synth",
-      syntheticSymbol: "SYNTH",
+      syntheticName: syntheticName,
```
Comment (nicholaspai) [inline packages/core/scripts/local/DeployEMP.js:110]: +1

File: packages/monitors/README.md [hunk 2]
```diff
@@ -10,6 +10,14 @@ For more information about running a monitor bot, see the [docs](https://docs.um
 yarn add @uma/monitors
 ```
 
+## Running the monitors
+
+The simplest way to run the monitors is:
+
+```bash
+EMP_ADDRESS=0x1234 CUSTOM_NODE_URL=https://your.node.url.io MNEMONIC="your mnemonic (12-word seed phrase) here" MONITOR_CONFIG="{your monitor config object}" monitors --network mainnet_mnemonic
```
Comment (nicholaspai) [inline packages/monitors/README.md:18]: What's the MVP MonitorConfig to get this to work?

Comment (pemulis) [reply packages/monitors/README.md:18]: Not sure yet! I should be able to put together an MVP config during the EMP war games, which will give me a better handle on the monitor bots.

Comment (pemulis) [reply packages/monitors/README.md:18]: Looks like a `MONITOR_CONFIG` is optional.

File: packages/core/scripts/local/DeployEMP.js [hunk 3]
```diff
@@ -26,9 +26,13 @@ const { interfaceName, ZERO_ADDRESS, parseFixed } = require("@uma/common");
 const { getAbi, getTruffleContract } = require("../../index");
 const argv = require("minimist")(process.argv.slice(), {
   boolean: ["test"],
-  string: ["identifier", "collateral", "cversion"]
+  string: ["identifier", "collateral", "cversion", "name", "symbol", "expiry"]
 });
 const abiVersion = argv.cversion || "1.2.2"; // Default to most recent mainnet deployment, 1.2.2.
+const syntheticName = argv.name || "Test Synth";
+const syntheticSymbol = argv.symbol || "SYNTH";
+const expiry = argv.expiry || 2;
```
Comment (nicholaspai) [inline packages/core/scripts/local/DeployEMP.js:34]:
```suggestion
const expiry = argv.expiry || 2; // Default to expire 2 mins from deployment
```

File: packages/core/scripts/local/DeployEMP.js [hunk 4]
```diff
@@ -26,9 +26,13 @@ const { interfaceName, ZERO_ADDRESS, parseFixed } = require("@uma/common");
...
+const expirationTimestamp = Math.ceil(Date.now() / 1000) + Number(expiry); // 2 minutes from now
```
Comment (nicholaspai) [inline packages/core/scripts/local/DeployEMP.js:35]:
```suggestion
const expirationTimestamp = Math.ceil(Date.now() / 1000) + Number(expiry);
```

File: packages/core/scripts/local/DeployEMP.js [hunk 5]
```diff
@@ -26,9 +26,13 @@ const { interfaceName, ZERO_ADDRESS, parseFixed } = require("@uma/common");
 const { getAbi, getTruffleContract } = require("../../index");
 const argv = require("minimist")(process.argv.slice(), {
   boolean: ["test"],
-  string: ["identifier", "collateral", "cversion"]
+  string: ["identifier", "collateral", "cversion", "name", "symbol", "expiry"]
 });
 const abiVersion = argv.cversion || "1.2.2"; // Default to most recent mainnet deployment, 1.2.2.
+const syntheticName = argv.name || "Test Synth";
```
Comment (nicholaspai) [inline packages/core/scripts/local/DeployEMP.js:32]: +1 for re organizing these constants up here

**Review by nicholaspai — APPROVED** [review]
Sweet changes LGTM

File: packages/core/scripts/local/DeployEMP.js (hunk 4 above)
Comment (daywiss) [reply packages/core/scripts/local/DeployEMP.js:35]: i think the naming of expiry is confusing, i think of it a date in the future, but its being used as a duration. Maybe might be better called duration or secondsTillExpiry or something that names the units so its a little more clear.

File: packages/core/scripts/local/DeployEMP.js (hunk 3 above)
Comment (daywiss) [reply packages/core/scripts/local/DeployEMP.js:34]: what units are these in? seconds or minutes

File: packages/core/scripts/local/DeployEMP.js (hunk 4 above)
Comment (daywiss) [reply packages/core/scripts/local/DeployEMP.js:35]: also if expiry is meant to be minutes it should be multiplied by 60

**Review by daywiss — COMMENTED** [review]
love the doc fixes, just want to make sure the math and expiry param is clearly understood

File: packages/core/scripts/local/DeployEMP.js (hunk 3 above)
Comment (pemulis) [reply packages/core/scripts/local/DeployEMP.js:34]: Good point, these should be minutes.

File: packages/core/scripts/local/DeployEMP.js (hunk 4 above)
Comment (pemulis) [reply packages/core/scripts/local/DeployEMP.js:35]: Yeah, this makes sense to me.

**Comment by pemulis** [comment]
I added a default to send all of the collateral or synthetic tokens from a particular address, which should help with sweeping tokens from any given bot or wallet into a central account that holds capital for liquidations, disputes, proposals, etc.

**Review by daywiss — APPROVED** [review]
looks good to me!

**Pull request merged**